lib/cocoapods/installer/xcode/pods_project_generator/target_installation_result.rb

```diff
@@ -58,12 +58,12 @@ class TargetInstallationResult
           # @param [Array<PBXNativeTarget>] test_native_targets @see #test_native_targets
           # @param [Hash{String=>Array<PBXNativeTarget>}] test_resource_bundle_targets @see #test_resource_bundle_targets
           # @param [Array<PBXNativeTarget>] test_app_host_targets @see #test_app_host_targets
-          # @param [Array<PBXNativeTarget>] app_native_targets @see #app_native_targets
+          # @param [Hash{Specification => PBXNativeTarget}] app_native_targets @see #app_native_targets
```

Comment:
should we update the ruby doc for the `attr_reader` of `app_native_targets` above?

Reply:
yep good catch